docs/docs-components/MarkdownPage.js

```diff
@@ -142,6 +144,31 @@ const components = {
   TwoCol: ({ children }: {| children: Node |}) => (
     <MainSection.Subsection columns={2}>{children}</MainSection.Subsection>
   ),
+  ImgContainer: ({ src, caption, alt }: {| src: string, caption?: string, alt?: string |}) => (
+    <Box>
+      <Box padding={8} rounding={2} borderStyle="sm" height="250px">
+        <Box position="relative" width="100%" height="100%">
+          <Image src={src} alt={alt} width="100%" height="100%" layout="fill" objectFit="contain" />
+        </Box>
+      </Box>
+      <Text size="300" align="start">
+        <Box as="figcaption" marginTop={3}>
+          {caption || ''}
```

Comment:
If `caption` is undefined, do we still want to render the Text and Box? Do we want a marginTop on no content?

Also: this looks like we instead want a Flex around the Image and Text with `gap={3}`. That gets the conditional margin for free.